composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.0",
         "fsi/datasource" : "~1.2",
-        "ruflin/elastica": "~1.2",
+        "ruflin/elastica": "~2.3",
```

Comment:
yea but composer does not care about that. If anybody used that ``0.x`` tag then composer update might breake his app. Even if you dump ``0.x`` to ``0.x+1``.